debugger.jpda/test/unit/src/org/netbeans/api/debugger/jpda/JPDASupport.java

```diff
@@ -241,7 +245,6 @@ public void doFinish () {
         debuggerEngine.getActionsManager ().
             doAction (ActionsManager.ACTION_KILL);
         waitState (JPDADebugger.STATE_DISCONNECTED);
-        deleteUserDir();
```

Comment:
Deleting the userdir at the end of the first test, while the module system is still running and some test execution is still pending does not work very well.